meta/Vertices.m

```diff
@@ -61,13 +61,32 @@
 
 Begin["`Private`"]
 
+FSVertex[sortedFields_, UseDependences_:False] := FSVertex[sortedFields, UseDependences] =
+Module[{vertexList, vertex},
+
+   vertexList =
+      Switch[Length[sortedFields],
+		   3, SARAH`VertexList3,
```

Comment:
Please check the indentation of these lines. :)

Reply:
Sorry Alex, but I don't see it. Is there a problem? It's consistently indented with 3 spaces.

Reply:
I can see it in the github diff of changes. I think it is due to a mixture of tabs and spaces: line 69 has 2 tabs and 4 spaces. Line 70 has 3 tabs. (Sorry, but it is actually just a minor and pedantic thing. :) )

Reply:
You're right. My bad. It's fixed.

Reply:
Thanks a lot!